### PR TITLE
Merging Development Branch

### PR DESCRIPTION
**Pull request by eduardoreisalvarenga**
We expect your comments. Thanks!

### CODE REVIEW

**Comment by samgaco** [comment]
### Hi @eduardoreisalvarenga and @davidauza-engineer :vulcan_salute: 

### You nailed this project, everything looks perfect :+1: 

**[REQUIRED]** Just **one thing to correct**, the elements in the footer should be arranged with either floats or flex.
**[SUGGESTION]** Review the section on[ git-flow](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59882084/) from the curriculum and apply it for the next projects :+1: 

### Just adjust for this required change and submit back for another code review :muscle:

File: style.css [hunk 1]
```diff
@@ -0,0 +1,955 @@
...
+.mobile-app-p {
+  margin-top: 14px;
+}
+
+/* Rules for the footer tag */
+footer {
+  margin-bottom: 86px;
+  text-align: center;
+}
+
```
Comment (samgaco) [inline style.css:892]: The footer elements should be arranged with either floats or display flex

**Review by samgaco — CHANGES_REQUESTED** [review]
### Hi @eduardoreisalvarenga and @davidauza-engineer :vulcan_salute: 

### You nailed this project, everything looks perfect :+1: 

**[REQUIRED]** Just **one thing to correct**, the elements in the footer should be arranged with either floats or flex.
**[SUGGESTION]** Review the section on[ git-flow](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59882084/) from the curriculum and apply it for the next projects :+1: 

### Just adjust for this required change and submit back for another code review :muscle:

File: style.css (hunk 1 above)
Comment (davidauza-engineer) [reply style.css:892]: Hi, the footer elements are now arranged using display: flex 💪

**Comment by davidauza-engineer** [comment]
All suggested changes are done 💪

**Comment by Torres-ssf** [comment]
### Pull Request Under Review By Sérgio (TSE)

**Review by Torres-ssf — APPROVED** [review]
### Nicely done team :clap: 
You can merge to master now :+1:
Good luck with the next one.

**Pull request merged**